docs/dub-reference/subpackages.md

```diff
@@ -6,7 +6,90 @@
 
     Tracking issue: <https://github.com/dlang/dub-docs/issues/71>
 
-For adding sub-packages from the registry as a dependency, execute `dub add <packagename>:<subpackage>`
+## Introduction
 
-If you need to compile a sub-package only, use `dub build :<subpackage>`
+DUB has support for sub-packages, a mechanic used to define additional optional
+recipes within the main recipe file. They can be built and tested separately.
```

Comment:
rather:

DUB has support for sub-packages which can be used to split your root package into multiple smaller ones which can be used individually within the same project, rather than needing to maintain and publish multiple packages on the registry. All sub-packages share the same versioning as the root package, but still act like regular individual packages on their own.